### PR TITLE
Prepare for a v0.3.0 release

### PR DESCRIPTION
**Pull request by kommendorkapten**
#### Summary
v0.3.0 release

#### Release Note
See updated CHANGLOG

#### Documentation

Rendered [CHANGELOG](https://github.com/kommendorkapten/protobuf-specs/blob/changelog-v3-b3/CHANGELOG.md)

### CODE REVIEW

**Comment by kommendorkapten** [comment]
Ignore the failing _generated code changes_ check now. There are two open PRs that should be merged first, so we have to regenerate it anyways before merging.

**Review by cpanato — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,20 +8,32 @@ All versions prior to 0.2.0 are untracked.
 
 ## [Unreleased]
 
-### Added
+### 0.3.0
```
Comment (haydentherapper) [inline CHANGELOG.md:11]:
```suggestion
## 0.3.0
```

**Comment by kommendorkapten** [comment]
@woodruffw yeah, we wait for #199 of course!

**Review by woodruffw — APPROVED** [review]
LGTM!

**Review by haydentherapper — APPROVED** [review]

**Pull request merged**